iothub/device/src/ClientPropertyCollection.cs

```diff
@@ -170,8 +170,9 @@ public bool Contains(string componentName, string propertyName)
         /// <typeparam name="T">The type to cast the object to.</typeparam>
         /// <param name="componentName">The component which holds the required property.</param>
         /// <param name="propertyName">The property to get.</param>
-        /// <param name="propertyValue">The value of the component-level property.</param>
-        /// <returns>true if the property collection contains a component level property with the specified key; otherwise, false.</returns>
+        /// <param name="propertyValue">When this method returns successfully, this contains the value of the component-level property.
```

Comment:
When this method returns ~~successfully/unsuccessfully~~ true/false ?

Reply:
I've used the same language as used in Dictionary.TryGetValue: https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.trygetvalue?view=net-5.0